src/java.desktop/share/classes/javax/swing/text/html/CSS.java

```diff
@@ -2965,6 +2965,17 @@ ImageIcon getImage(URL base) {
             }
             return image;
         }
+
+        @Override
+        public int hashCode() {
+            return (this.svalue != null) ? this.svalue.hashCode() : 0;
```

Comment:
This change look fine, but this line can be simplified by the Objects.hashCode();

Reply:
@prsadhuk idea was to replace whole expression with `Objects.hashCode`.
Current code looks confusing.